### PR TITLE
Add aditional visualization types

### PR DESCRIPTION
**Pull request by costinbusioc**
```
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX cr: <http://cr.eionet.europa.eu/ontologies/contreg.rdf#>
PREFIX a: <http://www.eea.europa.eu/portal_types/Data#>
PREFIX dcat: <http://www.w3.org/ns/dcat#>


SELECT DISTINCT ?type
WHERE {
   ?dataset a dcat:Dataset;
            a:relatedItems ?item.
   ?item rdf:type ?type.
} LIMIT 50
```
Using the above query, I identified the following related item types:
> http://rdfdata.eionet.europa.eu/amp/ontology/Output
> http://www.eea.europa.eu/portal_types/DavizVisualization#DavizVisualization
> http://www.eea.europa.eu/portal_types/Report#Report
> http://www.eea.europa.eu/portal_types/Data#Data
> http://www.w3.org/ns/dcat#Dataset
> http://www.eea.europa.eu/portal_types/Document#Document
> http://www.eea.europa.eu/portal_types/GIS%20Application#GISApplication
> http://www.eea.europa.eu/portal_types/EEAFigureFile#EEAFigureFile
> http://www.eea.europa.eu/portal_types/Promotion#Promotion
> http://www.eea.europa.eu/portal_types/File#File
> http://www.eea.europa.eu/portal_types/EEAFigure#EEAFigure)

The ticket mentions:
>     Dashboard (Tableau)
>     Interactive Map (GIS Application)
>     EEA Figure
>     Infographic

I don't really see a mapping for Infographic and Dashboard(Tableau). Also, the publish still returns an error 500 and I'm quite stuck in testing the workflow for now.

### CODE REVIEW

**Comment by costinbusioc** [comment]
Using the following query, I identified 661 possible types in the semantic repository.
```
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX cr: <http://cr.eionet.europa.eu/ontologies/contreg.rdf#>

SELECT DISTINCT ?p
WHERE {
  ?s rdf:type ?p.
} LIMIT 1000
```

Among the types, I found the ones requested in the ticket:
http://www.eea.europa.eu/portal_types/EEAFigure#EEAFigure
http://www.eea.europa.eu/portal_types/Dashboard#Dashboard
http://www.eea.europa.eu/portal_types/GIS%20Application#GISApplication
http://www.eea.europa.eu/portal_types/Infographic#Infographic

Currently, for each dataset, there is a related items property, leading to different visualisations (only Daviz is currently used).
Running this query:
```
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX cr: <http://cr.eionet.europa.eu/ontologies/contreg.rdf#>
PREFIX a: <http://www.eea.europa.eu/portal_types/Data#>
PREFIX dcat: <http://www.w3.org/ns/dcat#>


SELECT DISTINCT ?type
WHERE {
   ?dataset a dcat:Dataset;
            a:relatedItems ?item.
   ?item rdf:type ?type.
} LIMIT 50
```
I only identified the following types (of interest):
http://www.eea.europa.eu/portal_types/GIS%20Application#GISApplication
http://www.eea.europa.eu/portal_types/EEAFigure#EEAFigure
I guess that means GISApplication and EEAFigure represent forward related items?

In the ticket, there's a reference to: https://www.eea.europa.eu/data-and-maps/data/greenhouse-gas-emission-projections-for-6, which contains the following visualisation: <http://www.eea.europa.eu/data-and-maps/figures/ghg-emissions-trends-and-projections>

Running:
```
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX cr: <http://cr.eionet.europa.eu/ontologies/contreg.rdf#>
SELECT *
WHERE {
  <http://www.eea.europa.eu/data-and-maps/figures/ghg-emissions-trends-and-projections> ?p <http://www.eea.europa.eu/data-and-maps/data/greenhouse-gas-emission-projections-for-6>.
} LIMIT 5000
```
we obtain the property: http://www.eea.europa.eu/portal_types/EEAFigure#relatedItems

If I understand correctly, does that mean, a dataset will be forwared linked only to Daviz/EEAFigure/GISApplication, whereas Infographic, Dashboard (and EEAFigure) will be backward related items to the dataset?

**Comment by costinbusioc** [comment]
Example of updated pages:
1. https://webgate.acceptance.ec.testa.eu/euodp/data/dataset/DAT-2-en (contains forward Dashboard and EEA Figure)
1. https://webgate.acceptance.ec.testa.eu/euodp/data/dataset/DAT-150-en (contains backward Infographic and backward EEA Figure)
1. https://webgate.acceptance.ec.testa.eu/euodp/data/dataset/DAT-176-en (contains backward Dashboard)

At the moment there are only two GIS Applications visualisations, both of them being expired

File: app/tests/test_odp_related_items.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    assert g.value(dist_daviz, DCAT.accessURL) == URIRef(daviz_url)
+
+
+def test_query_retrieve_forward_gis_application(mocker):
+    """
+        TODO: At the moment there are only two GIS Applications visualisations, both of
+        them being expired. The SPARQL queries check for visualisations with no
+        expiration dates. When a relevant example will exist, a test should be implemented.
+
+        The query that displays all the GIS Application visualisations alongside their expirations is:
```
Comment (mgax) [inline app/tests/test_odp_related_items.py:110]: That's some neat documentation, nice!

**Review by mgax — APPROVED** [review]
Looks great, especially the tests!

**Pull request merged**